CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.5.6
+
+* Update tax and benefit system initialization method in scripts.
```

Comment:
I don't think this changes anything for the user, if I understand well @cbenz repaired untested scripts that were not adjusted to new version of `france` a few months ago.